ortools/dotnet/Google.OrTools/Google.OrTools.csproj.in

```diff
@@ -1,234 +1,105 @@
+    </Compile>
   </ItemGroup>
 
-  <!-- Check at least one runtime package has been generated -->
+  <!-- Validate at least one runtime package has been generated -->
   <Target Name="ValidateRuntimePack" BeforeTargets="GenerateNuspec">
     <ItemGroup>
-      <RuntimeLinux Include="../../../packages/Google.OrTools.runtime.linux-x64.*.nupkg" />
-      <RuntimeOsx Include="../../../packages/Google.OrTools.runtime.osx-x64.*.nupkg" />
-      <RuntimeWin Include="../../../packages/Google.OrTools.runtime.win-x64.*.nupkg" />
+      <RuntimeLinux Include="../../../packages/Google.OrTools.runtime.linux-x64.*.nupkg"/>
+      <RuntimeOsx Include="../../../packages/Google.OrTools.runtime.osx-x64.*.nupkg"/>
+      <RuntimeWin Include="../../../packages/Google.OrTools.runtime.win-x64.*.nupkg"/>
     </ItemGroup>
-    <Message
-      Importance="high"
-      Text="Package Google.OrTools.runtime.linux-x64: found"
-      Condition="Exists('@(RuntimeLinux)')" />
-    <Message
-      Importance="high"
-      Text="Package Google.OrTools.runtime.linux-x64: not found"
-      Condition="!Exists('@(RuntimeLinux)')" />
-    <Message
-      Importance="high"
-      Text="Package Google.OrTools.runtime.osx-x64: found"
-      Condition="Exists('@(RuntimeOsx)')" />
-    <Message
-      Importance="high"
-      Text="Package Google.OrTools.runtime.osx-x64: not found"
-      Condition="!Exists('@(RuntimeOsx)')" />
-    <Message
-      Importance="high"
-      Text="Package Google.OrTools.runtime.win-x64: found"
-      Condition="Exists('@(RuntimeWin)')" />
-    <Message
-      Importance="high"
-      Text="Package Google.OrTools.runtime.win-x64: not found"
-      Condition="!Exists('@(RuntimeWin)')" />
-    <Error
-      Text="You must locally 'dotnet pack' at least one Google.OrTools.runtime.{rid} Project."
-      Condition="!Exists('@(RuntimeLinux)') AND !Exists('@(RuntimeOsx)') AND !Exists('@(RuntimeWin)')" />
+    <Message Importance="high" Text="Package Google.OrTools.runtime.linux-x64: found" Condition="Exists('@(RuntimeLinux)')"/>
+    <Message Importance="high" Text="-------  $(OS)  "/>
+    <Message Importance="high" Text="Package Google.OrTools.runtime.linux-x64: not found" Condition="!Exists('@(RuntimeLinux)')"/>
+    <Message Importance="high" Text="Package Google.OrTools.runtime.osx-x64: found" Condition="Exists('@(RuntimeOsx)')"/>
+    <Message Importance="high" Text="Package Google.OrTools.runtime.osx-x64: not found" Condition="!Exists('@(RuntimeOsx)')"/>
+    <Message Importance="high" Text="Package Google.OrTools.runtime.win-x64: found" Condition="Exists('@(RuntimeWin)')"/>
+    <Message Importance="high" Text="Package Google.OrTools.runtime.win-x64: not found" Condition="!Exists('@(RuntimeWin)')"/>
+    <Error Text="You must locally 'dotnet pack' at least one Google.OrTools.runtime.{rid} Project." Condition="!Exists('@(RuntimeLinux)') AND !Exists('@(RuntimeOsx)') AND !Exists('@(RuntimeWin)')"/>
   </Target>
 
-  <!-- Copy all .proto file(s) -->
```

Comment:
Are they still provided ? This is a user's request.

Reply:
No, they are not part of the Nuget packages anymore. The problem is that these .proto files are pulled into the consumer's project root directory during Nuget installation. Which results in a lot of files and folders in it's project system, which are not needed by 99,9% of the users. It would be possible to create a separate, completely independent Nuget, with the proto files. But this I can not do into this PR, because I am a little bit in a hurry (tomorrow I will be in holidays for 21 days). Maybe I could revert this change, what do you think?